src/context.cpp

```diff
@@ -9,6 +9,11 @@
 #include "dds/rsdds-device-factory.h"
 #endif
 
+#include <librealsense2/hpp/rs_types.hpp>  // rs2_devices_changed_callback
+#include <librealsense2/rs.h>              // RS2_API_VERSION_STR
```

Comment:
I find it helps -- at least for this version of the code.
But you're right, it likely won't get updated.
Hopefully I'm getting the context to be focused-enough so that it'll never have to be updated again...